kotlin/src/main/java/app/rive/runtime/kotlin/core/Artboard.kt

```diff
@@ -187,8 +186,8 @@ class Artboard(val cppPointer: Long) {
     /**
      * Draw the the artboard to the [renderer].
      */
-    fun drawSkia(rendererAddress: Long, fit: Fit, alignment: Alignment) {
-        cppDrawSkia(cppPointer, rendererAddress, fit, alignment)
+    fun drawSkia(rendererAddress: Long) {
+        cppDrawSkia(cppPointer, rendererAddress)
```

Comment:
So, yeah, we could have both functions here:
```kotlin
fun drawSkia(rendererAddress: Long, fit: Fit, alignment: Alignment)
fun drawSkia(rendererAddress: Long)
```
With the only caveat that the underlying JNI call must have a unique name (I think).